src/test/java/org/kiwiproject/jersey/client/dropwizard/DropwizardManagedClientBuilderTest.java

```diff
@@ -175,4 +177,64 @@ void shouldBuildRegistryAwareClient() {
             assertThat(client).isInstanceOf(RegistryAwareClient.class);
         }
     }
+
+    @Nested
+    class NewDefaultJerseyClientConfiguration {
+
+        @Test
+        void shouldReturnNewJerseyClientConfiguration_WithoutTLS_WhenProviderNotProvided() {
```

Comment:
Since there are multiple assertions in each of these, suggest soft asserting them all...